tests/cli/test_build.py

```diff
@@ -196,6 +196,7 @@ def test_build_buildspecs(self):
             exclude_buildspecs=[excluded_buildspecs[0]],
             buildtest_system=self.system,
         )
+        cmd = Buildtest(...)
```

Comment:
i dont mean you literally copy the suggestion. The `...` was meant for you to fill the appropriate arguments to the class so you can invoke the BuildTest class. Please see examples from above. Once you are ready for review please ping me. In the meantime, i will mark this as a `Draft`